giskard/models/langchain.py

```diff
@@ -38,15 +38,32 @@ def __init__(
             **kwargs,
         )
 
-    def save_model(self, local_path, mlflow_meta):
-        mlflow.langchain.save_model(self.model, path=local_path, mlflow_model=mlflow_meta)
+    def save(self, local_path: Union[str, Path]) -> None:
+        super().save(local_path)
```

Comment:
Well actually `save_model` isn't called in neither `WrapperModel` nor `BaseModel`

Reply:
Oh, you're right, my bad. I relied on my memory instead of looking at the code. I thought we had the same logic as `model_predict` and `predict`, but indeed we kept `save` loose I think because of the ordering of the saving.